_docs/concepts/network-and-auth/auth.md

```diff
@@ -16,34 +16,35 @@ Istio Auth's aim is to enhance the security of microservices and their communica
 
 *   Providing each service with a strong identity that represents its role to enable interoperability across clusters and clouds
 
-*   Securing service to service communication
+*   Securing service to service communication and end-user to service communication
 
 *   Providing a key management system to automate key and certificate generation, distribution, rotation, and revocation
 
-In future versions it will also provide:
+*   Upcoming features:
+    *   Powerful authorization mechanisms: [ABAC](https://docs.google.com/document/d/1U2XFmah7tYdmC5lWkk3D43VMAAQ0xkBatKmohf90ICA/edit), [RBAC](https://docs.google.com/document/d/1dKXUEOxrj4TWZKrW7fx_A-nrOdVD4tYolpjgT8DYBTY/edit), Authorization hooks.
+    *   [End-user authentication](https://docs.google.com/document/d/1rU0OgZ0vGNXVlm_WjA-dnfQdS3BsyqmqXnu254pFnZg/edit)
```

Comment:
not sure if having links into google docs requiring signin and access request is very good for end users browsing the istio.io website

Reply:
These docs are on istio/community drive, so the community should be able to view them.